content/en-us/reference/engine/classes/RotationCurve.yaml

```diff
@@ -5,8 +5,7 @@ memory_category: Instances
 summary: |
   Represents a sequence of rotations and the interpolation curve between them.
 description: |
-  This class holds a sorted list of `RotationCurveKey|RotationCurveKeys` that
-  represent a sequence of rotations. The shape of the interpolation curve
+  This class holds a sorted list of `Class.RotationCurveKey|RotationCurveKeys` that represent a sequence of rotations. The shape of the interpolation curve
```

Comment:
Not sure what's the deal with RotationCurveKey. It's not a class that appears in the docs, but it's also not manually hidden, at least as far as I can tell. So presumably it's a data type? Those pages need to be manually created in the docs system; they don't come from the API dump.

Reply:
Yep, it's a DataType. Page should be created.... then this link will work.

Reply:
Who knows the details on RotationCurveKey? I see it in the dump, but only cited as parameters or returns of other things. Do you have access to Engine code? If so, I think you may be able to inspect the C code there and perhaps discern who can tell us what RotationCurveKey is and how to document it.

Reply:
I'm guessing it will basically be the same as https://create.roblox.com/docs/reference/engine/datatypes/FloatCurveKey , they have all the same properties in studio's autocomplete